concourse/tasks/deploy-to-govuk-paas.yml

```diff
@@ -38,8 +38,8 @@ run:
       cf api "$CF_API"
       cf auth
       cf t -o "$CF_ORG" -s "$CF_SPACE"
-      cf v3-create-app govuk-coronavirus-find-support || true
-      cf v3-apply-manifest -f manifest.yml
+      cf push govuk-coronavirus-find-support --strategy rolling || true
+      cf apply-manifest -f manifest.yml
```

Comment:
I think it would be closer to the original to use `cf create-app` (note: dropping the `v3-` prefix), and then to replace `cf v3-zdt-push` with `cf push --strategy rolling`.

As a bit of context, the `cf create-app` and `cf apply-manifest` commands are like little bits of what happens when you run `cf push` (which tries to do everything at once).

Ideally, we wouldn't have all this scripting and we'd just have a single `cf push` command with a manifest that contained everything we needed. I couldn't make that work the way I wanted before - if I remember correctly the issue was that I needed to set a bunch of environment variables to secret values, and `cf v3-zdt-push` wouldn't let me pass them in on the command line, so I had to set them separately with `cf set-env`.

I _think_ now that we're on the newer CLI we could fix a lot of the tech debt here, and just state explicitly in the manifest file which variables we need to provide.

I'd suggest experimenting in a local terminal pushing to a sandbox space would be a good idea though.

Reply:
Thanks for the advice @richardTowers! I've made the suggested changes and fixed up. Now to find a sandbox to test it...